perf/stability/control-plane/istiod.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: apps/v1
```

Comment:
I think this autogenerated or something? Not clear how its used?

Reply:
I was testing with the different limits and requests that's why created a separate istiod.yaml. I don't think we would need this in the commit, will update that. Thanks!!